src/System.Threading.Tasks.Extensions/ref/System.Threading.Tasks.Extensions.cs

```diff
@@ -24,20 +25,22 @@ public partial struct AsyncValueTaskMethodBuilder<TResult>
         public void SetStateMachine(System.Runtime.CompilerServices.IAsyncStateMachine stateMachine) { }
         public void Start<TStateMachine>(ref TStateMachine stateMachine) where TStateMachine : System.Runtime.CompilerServices.IAsyncStateMachine { }
     }
-    public partial struct ConfiguredValueTaskAwaitable<TResult>
+    public readonly partial struct ConfiguredValueTaskAwaitable<TResult>
```

Comment:
@stephentoub while regenerating this a few things got marked as readonly I assume that is more leftover from your other work.  In this ref `ConfiguredValueTaskAwaiter` and `ValueTaskAwaiter<TResult>` also got marked as readonly based on what the netstandard [implementation](https://github.com/dotnet/corefx/blob/45b724f6b6391910edea8a70f3f22a4a7996696d/src/System.Threading.Tasks.Extensions/src/System/Runtime/CompilerServices/ConfiguredValueTaskAwaitable.cs#L37) is but they aren't marked that way in [corelib](https://github.com/dotnet/coreclr/blob/19d9bd09d4da47d77e9e94bcd965499c82d8a0df/src/mscorlib/shared/System/Runtime/CompilerServices/ConfiguredValueTaskAwaitable.cs#L38), so I undid it in the ref. Please feel free to make the consistent as you deem fit.